src/AppInstallerRepositoryCore/PackageDependenciesValidationUtil.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This header is used outside of the RepositoryCore project, so it needs to be put under Public/winget and exposed there

And imo, this is easily expandable to other ManifestException types in the future, we could probably put it in CommonCore beside ManifestValidation.h, or just declare the function in  ManifestValidation.h